docs/Agent/user-guide/man_page.md

```diff
@@ -2,32 +2,32 @@
  
 **Contents**
 
-#### Performance tool management commands
+## Performance tool management commands
 -  [pbench-clear-results](#pbench-clear-results)
 -  [pbench-clear-tools](#pbench-clear-tools)
 -  [pbench-list-tools](#pbench-list-tools)
 -  [pbench-list-triggers](#pbench-list-triggers)
 -  [pbench-register-tool](#pbench-register-tool)
 -  [pbench-register-tool-set](#pbench-register-tool-set)
 -  [pbench-register-tool-trigger](#pbench-register-tool-trigger)
-#### Benchmark commands
+## Benchmark commands
 -  [pbench-user-benchmark](#pbench-user-benchmark)
-#### Upload to Pbench Server
-###### Pbench Server 0.69
+## Upload to Pbench Server
+### Pbench Server 0.69
 - [pbench-move-result](#pbench-move-result)
 - [pbench-copy-results](#pbench-copy-results) 
-###### Pbench Server 1.0
+### Pbench Server 1.0
 -  [pbench-results-move](pbench-results-move)
 
 
 ---
 
-##### pbench-clear-results
+#### pbench-clear-results
```

Comment:
I didn't want to go too far in arguing for a strictly correct logical nesting, especially since it appears at casual inspection that GitHub doesn't really do anything with the logical nesting aside from minor formatting differences...

However, I did some investigation and I found out that GitHub automatically generates a real nested TOC list when you view the page (icon at the top right), and it clearly shows the unusual nesting, which I don't like at all:

![image](https://github.com/distributed-system-analysis/pbench/assets/61761529/e0bd2399-83af-4f41-983c-3111501f0435)

I think that, logically, maybe something like changing the `**Contents**` line to `## Commands by functional group` with `### Performance tool management commands` nested under that; and then `## Commands in alphabetical order`, with the commands underneath that at level 3?

Reply:
So, where now we have

```
# Man pages
Contents
## (...) management commands
## benchmark commands
[...]
## Upload
### 0.69
### 1.0
#### pbench-clear-results
```

We might instead have

```
# Man pages
## Contents by function
### (...) management commands
#### benchmark commands
[...]
#### Upload
##### 0.69
##### 1.0
## Commands
### pbench-clear-results
```